lib/Cake/TestSuite/CakeTestCase.php

```diff
@@ -68,52 +73,6 @@ abstract class CakeTestCase extends \PHPUnit\Framework\TestCase {
  */
 	protected $_pathRestore = array();
 
-/**
- * Runs the test case and collects the results in a TestResult object.
- * If no TestResult object is passed a new one will be created.
- * This method is run for each test method in this class
- *
- * @param TestResult $result The test result object
- * @return TestResult
- * @throws InvalidArgumentException
- */
-	public function run(TestResult $result = null): TestResult {
-		$level = ob_get_level();
-
-		if (!empty($this->fixtureManager)) {
-			$this->fixtureManager->load($this);
-		}
-		$result = parent::run($result);
-		if (!empty($this->fixtureManager)) {
-			$this->fixtureManager->unload($this);
-			unset($this->fixtureManager, $this->db);
-		}
-
-		for ($i = ob_get_level(); $i < $level; ++$i) {
-			ob_start();
```

Comment:
I don't know what this ob_* stuff was for, but deleting it doesn't break anything. Cake also has a browser test runner, so it might have been for that.